Makefile

```diff
@@ -454,6 +456,13 @@ base-go.docker: Dockerfile.base-go $(var.)BASE_GO_IMAGE $(WRITE_IFCHANGED)
 	fi
 	@docker image inspect $(BASE_GO_IMAGE) --format='{{.Id}}' | $(WRITE_IFCHANGED) $@
 
+envoy-tests-image:
```

Comment:
Should this be recast to use `docker.mk`, now that we have it?

Reply:
I don't fully understand how that works, but I've made a few changes to accommodate this, lmk :)